ui/app/templates/components/secret-create-or-update.hbs

```diff
@@ -160,22 +164,29 @@
   >
     <div class="box is-sideless is-fullwidth is-marginless padding-top">
       <MessageError @model={{@modelForData}} @errorMessage={{this.error}} />
-      {{#if (eq @canReadSecretData false)}}
-        <AlertBanner
-          @type="warning"
-          @message="You do not have read permissions. If a secret exists here creating a new secret will overwrite it."
-          data-test-warning-no-read-permissions
-        />
-      {{/if}}
       <NamespaceReminder @mode="edit" @noun="secret" />
-      {{#if this.isCreateNewVersionFromOldVersion}}
-        <div class="form-section">
-          <AlertBanner
-            @type="warning"
-            @class="is-marginless"
-            @message="You are creating a new version based on data from Version {{@model.selectedVersion.version}}. The current version for {{@model.id}} is Version {{@model.currentVersion}}."
-          />
-        </div>
+      {{#if (or (eq @canReadSecretData false) this.isCreateNewVersionFromOldVersion)}}
```

Comment:
combined banners into one message (pattern we used in client count dashboard)
<img width="853" alt="Screenshot 2023-06-13 at 4 23 44 PM" src="https://github.com/hashicorp/vault/assets/68122737/30d43774-b4c7-4a25-b9c1-007ce1c1c49b">